### ISSUE TITLE
'NoneType' object has no attribute 'attrs'

### ISSUE THREAD

**Issue by jayvdb** [django-import-export/django-import-export#1163]
I'm trying import-export on Django 3, python 3.8, on a fairly large site with lots of models, but I encounter this on the first class inspected in this code, which is `import_export.resources.Resource`.

```py
    from import_export import resources
  File "/usr/lib/python3.8/site-packages/import_export/resources.py", line 234, in <module>
    class Resource(metaclass=DeclarativeMetaclass):
  File "/usr/lib/python3.8/site-packages/import_export/resources.py", line 202, in __new__
    setattr(meta, option, getattr(options, option))
AttributeError: 'NoneType' object has no attribute 'attrs'
```

When I add code to avoid that (`if options:`), I then encounter

```py
    from import_export import resources
  File "/usr/lib/python3.8/site-packages/import_export/resources.py", line 986, in <module>
    class ModelResource(Resource, metaclass=ModelDeclarativeMetaclass):
  File "/usr/lib/python3.8/site-packages/import_export/resources.py", line 914, in __new__
    new_class = super().__new__(cls, name, bases, attrs)
  File "/usr/lib/python3.8/site-packages/import_export/resources.py", line 183, in __new__
    setattr(meta, option, getattr(options, option))
AttributeError: 'NoneType' object has no attribute 'attrs'
```

Again a guard `if options:` around that code helps.

 - Django Import Export: 2.0.2, 2.3.0, master
 - Python 3.8
 - Django 3.0.8

Odd packages which might be related are `django-compat-patcher`, and `forbiddenfruit`.

**Comment by andrewgy8** [comment]
Interesting. Do you think you can create a reproducible example? Im wondering if its a django-import-export issue, or maybe as you point out, a conflict with another package.

**Comment by juanjux** [comment]
We also found this problem, in our case the conflicting module was definitely forbiddenfruit, just importing it shows the problem, we added `if options and meta:` like @jayvdb and it worked trough eventually we did a change to forbiddenfruit since we have it vendored.

**Comment by jayvdb** [comment]
@juanjux , is your `forbiddenfruit` modification public?  Seeing it might highlight, or at least help narrow down, the cause.

**Comment by juanjux** [comment]
On `forbiddenfruit.py` we replaced:

```
calling_frame = inspect.currentframe().f_back
return sorted(calling_frame.f_locals.keys())
```

with:

```
return  __builtin__.dir(obj)
```

So it's doing a dir(None) but that's ok for our use case of the module.

**Comment by matthewhegarty** [comment]
It's easy to reproduce when testing `django-import-export`

1. `pip install forbiddenfruit`
2.  in `test_resources.py`, add `import forbiddenfruit`
3. Run tests

tests will crash with the error message.

**Comment by matthewhegarty** [comment]
Fixed in #1325
